tests/liquidation_test.py

```diff
@@ -1,4 +1,74 @@
 import unittest
+from web3 import Web3
+
+from mev_inspect.aave_liquidations import get_liquidations
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.classified_traces import Protocol
+from mev_inspect.classifiers.trace import TraceClassifier
+from mev_inspect.block import create_from_block_number
+
+
+class TestAaveLiquidations(unittest.TestCase):
```

Comment:
check out how we use pytest in the arbitrage integration tests instead of unit tests

they're a simpler / don't require a class